.env

```diff
@@ -1,9 +1,9 @@
-MYSQL_ROOT_USERNAME=root
```

Comment:
create .env.template file, remove values, add .env to gitignore and remove it from git